src/pages/docs/chat/rooms/index.mdx

```diff
@@ -7,28 +7,29 @@ Rooms are used to organize and logically separate your users and chat messages i
 
 ## Relationship between rooms and channels <a id="channel-relationship"/>
 
-It is important to note that each room is backed by a single Ably Pub/Sub channel. You may need to be aware of this when using some integrations with Ably, such as the [Pulsar](/docs/platform/integrations/streaming/pulsar#creating-pulsar-rule) connectors, or if you are operating in an environment where a Chat SDK is not available.
+It is important to note that each room is backed by a single Ably Pub/Sub channel. You may need to be aware of this when using some [integrations](/docs/chat/integrations) with Ably, such as the [Pulsar](/docs/platform/integrations/streaming/pulsar#creating-pulsar-rule) connectors, or if you are operating in an environment where a Chat SDK is not available.
 
 The channel name is the same as the room name with an appended suffix of `::$chat` (e.g `some-room-id::$chat`). In most cases, you will not need to worry about this, as the Chat SDK handles the channel creation and management for you and capabilities can be configured at the room level.
 
-## Create or retrieve a room <a id="create"/>
+## Get a reference to a room <a id="create"/>
 
 Users send messages to a room and subscribe to the room in order to receive messages.
 
 <If lang="javascript,swift,kotlin">
-A `room` is created, or an existing one is retrieved from the `rooms` collection using the <If lang="javascript">[`rooms.get()`](https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat-js.Rooms.html#get)</If><If lang="swift">[`rooms.get()`](https://sdk.ably.com/builds/ably/ably-chat-swift/main/AblyChat/documentation/ablychat/rooms/get%28named%3Aoptions%3A%29)</If><If lang="kotlin">[`rooms.get()`](https://sdk.ably.com/builds/ably/ably-chat-kotlin/main/dokka/chat/com.ably.chat/-rooms/get.html)</If> method:
+To get a reference to a chat room, use the <If lang="javascript">[`rooms.get()`](https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat-js.Rooms.html#get)</If><If lang="swift">[`rooms.get()`](https://sdk.ably.com/builds/ably/ably-chat-swift/main/AblyChat/documentation/ablychat/rooms/get%28named%3Aoptions%3A%29)</If><If lang="kotlin">[`rooms.get()`](https://sdk.ably.com/builds/ably/ably-chat-kotlin/main/dokka/chat/com.ably.chat/-rooms/get.html)</If> method. It will create a new room reference if one doesn't already exist, or return the existing one if it does.
```

Comment:
"Reference" implies something already exists - is that the right word here?

Reply:
I wanted to avoid wording that suggests a room is created (so avoid "a new room"). I find it more accurate that we create a reference to a room, or a room client (I wouldn't say that though) or similar.

Reference was the best I could think of?

If it creates more confusion than before maybe just go back to "a new room" type of wording?

WDYT?

Reply:
I like `instance`. All updated, thanks!